Customization/T220/PhoneRepairShop_Code/PhoneRepairShop_Code/PhoneRepairShop_code.csproj

```diff
@@ -81,13 +76,21 @@
     <Compile Include="DAC\RSSVLabor.cs" />
     <Compile Include="DAC\RSSVWarranty.cs" />
     <Compile Include="RSSVWorkOrderEntry.cs" />
-    <Compile Include="DAC\RSSVWorkOrder.cs" />
     <Compile Include="DAC\RSSVWorkOrderItem.cs" />
     <Compile Include="DAC\RSSVWorkOrderLabor.cs" />
     <Compile Include="RSSVSetupMaint.cs" />
     <Compile Include="DAC\RSSVSetup.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
I'm not sure we have these references in other T course projects. Are they required?